# Detected object array
DetectedObject[] objects
